raiden-network/light-client
# dApp: blockchain progress for opening a channel

Fixes #1941

**Short description**
Introduces notification for opening a channel with a block counter that counts until channel gets opened.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Open a channel. Snackbar should appear.
2. Open the notification panel, a notification for the open channel event should be shown with a counter.
